Resources/de.lproj/ModalSheets.xib.strings

```diff
@@ -62,8 +62,8 @@
 "1026.title" = "Abbrechen";
 "1095.title" = "Abbrechen";
 "1156.title" = "Erweiterte Optionen";
-"1358.title" = "Upload public key";
+"1358.title" = "Öffentlichen Schlüssel hochladen";
 "FWA-34-yOE.title" = "Abbrechen";
 "Tbh-Pz-SCs.title" = "Label";
-"c3R-Qv-JF7.title" = "GPG Keychain will move your secret keys to the selected location. gpg will find your keys as soon as the target volume is connected to this machine.\n\nIMPORTANT: After moving your secret keys, you will always need the new target volume to sign or decrypt mails or files.";
-"eKE-tf-ktq.title" = "Willst du das deine Signatur verfällt (empfohlen)?";
+"c3R-Qv-JF7.title" = "GPG Schlüsselbund wird deinen geheimen Schlüssel an den gewählten Ort bewegen. gpg wird deine Schlüssel finden, sobald das Zielvolumen mit dem Computer verbunden wurde. \n\nWICHTIG: Nachdem die geheimen Schlüssel verschoben wurden, wirst du immer das Zielvolume brauchen um Mails oder Dateien zu signieren oder zu entschlüsseln.";
```

Comment:
I'd suggest "Laufwerk" instead of "Volumen". Also the second "gpg" should probably be all uppercase.